models/neurons/aeif_psc_alpha_neuron.nestml

```diff
@@ -0,0 +1,119 @@
+"""
+aeif_psc_alpha - Conductance based exponential integrate-and-fire neuron model
+###############################################################################
+
+Description
++++++++++++
+
+aeif_psc_alpha is the adaptive exponential integrate and fire neuron according to Brette and Gerstner (2005), with post-synaptic conductances in the form of a bi-exponential ("alpha") function.
+
+The membrane potential is given by the following differential equation:
+
+.. math::
+
+   C_m \frac{dV_m}{dt} =
+   -g_L(V_m-E_L)+g_L\Delta_T\exp\left(\frac{V_m-V_{th}}{\Delta_T}\right) -
+ g_e(t)(V_m-E_e) \\
+                                                     -g_i(t)(V_m-E_i)-w + I_e
+
+and
+
+.. math::
+
+ \tau_w \frac{dw}{dt} = a(V_m-E_L) - w
+
+Note that the membrane potential can diverge to positive infinity due to the exponential term. To avoid numerical instabilities, instead of :math:`V_m`, the value :math:`\min(V_m,V_{peak})` is used in the dynamical equations.
+
+
+References
+++++++++++
+
+.. [1] Brette R and Gerstner W (2005). Adaptive exponential
+       integrate-and-fire model as an effective description of neuronal
+       activity. Journal of Neurophysiology. 943637-3642
+       DOI: https://doi.org/10.1152/jn.00686.2005
+
+
+See also
+++++++++
+
+iaf_psc_alpha, aeif_psc_exp
+"""
+model aeif_psc_alpha_neuron:
+
+    state:
+        V_m mV = E_L       # Membrane potential
+        w pA = 0 pA        # Spike-adaptation current
+        refr_t ms = 0 ms    # Refractory period timer
+        is_refractory integer = 0
+        g_exc nS = 0 nS      # AHP conductance
+        g_exc' nS/ms = 0 nS/ms   # AHP conductance
+        g_inh nS = 0 nS      # AHP conductance
+        g_inh' nS/ms = 0 nS/ms   # AHP conductance
+
+    equations:
+        inline V_bounded mV = min(V_m, V_peak) # prevent exponential divergence
+
+        g_exc'' = -2 * g_exc' / tau_exc - g_exc / tau_exc**2
+        g_inh'' = -2 * g_inh' / tau_inh - g_inh / tau_inh**2
+
+        # Add inlines to simplify the equation definition of V_m
+        inline exp_arg real = (V_bounded - V_th) / Delta_T
+        inline I_spike pA = g_L * Delta_T * exp(exp_arg)
+        inline I_syn_exc pA = g_exc / nS * pA
```

Comment:
could the documentation, variable naming convention be cleaned up and these two lines (that have the expression ``/nS*pA``) be removed?